### DISCUSSION TITLE
表中字段数据实际长度总是比字符串实际长度要长是因为什么原因

### DISCUSSION THREAD

**Discussion by linzhicong1996** [apache/doris#23890] [A - General / Q&A]
想请教下大佬这个字符长度的计算规则，实际上已经从32扩展到40了
![image](https://github.com/apache/doris/assets/94167908/0f2e18d4-4538-4b18-82ea-bc428a5a8a72)
![image](https://github.com/apache/doris/assets/94167908/d6985032-26c1-4f0e-b2a4-d14e08f1b9e0)

**Comment by linzhicong1996** [comment]
![image](https://github.com/apache/doris/assets/94167908/257fc0e9-ae00-45cb-bec7-7f49b2b473df)

**Comment by LemonLiTree** [comment]
<img width="723" alt="image" src="https://github.com/apache/doris/assets/109049295/8b99b175-143c-4e1f-acff-ee0f593e9418">
Which version did you test on? My test is fine

**Reply by linzhicong1996** [reply]
sorry.Mistakes made due to my carelessness .valuse has '\u200b' and doris varchar Field length is string in byte size .And i am forget close question.I genuinely appreciate your help and support.
